### PR TITLE
ASSET: Deleted the instances of `MVP-04` references from the Contributing.md

### LINKED ISSUE #1558: Update the CONTRIBUTING.md doc to remove references to MVP-04
### Overview
We are no longer using MVP-04

### Action Items
 - [x] Remove instructions for installation and folder generation of `client/mvp-04`

### Resources/Instructions
[File here](https://github.com/hackforla/VRMS/blob/development/CONTRIBUTING.md#part-2-how-to-set-up-the-development-environment)

### PR DESCRIPTION
**Pull request by freaky4wrld**
Fixes #1558 

### What changes did you make and why did you make them ?

  - The instances to of `MVP-04` are removed from the instructions point directing to add the required variables for `frontend and backend` and for installing the node packages in `MVP-04` directory
  - This is done as we have terminated `MVP-04` folder from the project file-system and we aren't using it as of the current scenario
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
- No visual changes

### CODE REVIEW

**Review by jbubar — APPROVED** [review]
This looks great! Thanks Nayan!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -114,14 +114,12 @@ Note: Understanding how git remotes work will make collaborating much easier. Yo
...
-   - `cd client-mvp-04` and run `yarn install`
    - `cd ../backend` and run `yarn install`
 
 1. Add your required environment variables for the frontend and backend directories:
 
    - `touch vrms/backend/.env`
    - `touch vrms/client/.env`
-   - `touch vrms/client-mvp-04/.env`
 
    Note 1: In the above example you are trying to create an empty file called `.env` in each of the listed directories: backend, client and client-mvp-04. You can use either `touch <path-to-directory> .env` or navigate to the directory and use `touch .env`
```
Comment (trillium) [inline CONTRIBUTING.md]: Change section

```md
Note 1: In the above example you are trying to create an empty file called .env in each of the listed directories: backend, client and client-mvp-04. You can use either touch <path-to-directory> .env or navigate to the directory and use touch .env
```

to

```md
Note 1: In the above example you are trying to create an empty file called .env in each of the listed directories: backend and client. You can use either touch <path-to-directory> .env or navigate to the directory and use touch .env
```

**Review by trillium — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (trillium) [reply CONTRIBUTING.md]: The changed block is `backend, client and client-mvp-04` -> `backend and client`

Comment (freaky4wrld) [reply CONTRIBUTING.md]: The needed changes are done, thanks for mentioning the unnoticed instance

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,16 +114,14 @@ Note: Understanding how git remotes work will make collaborating much easier. Yo
...
    - `cd ../backend` and run `yarn install`
 
 1. Add your required environment variables for the frontend and backend directories:
 
    - `touch vrms/backend/.env`
    - `touch vrms/client/.env`
-   - `touch vrms/client-mvp-04/.env`
 
-   Note 1: In the above example you are trying to create an empty file called `.env` in each of the listed directories: backend, client and client-mvp-04. You can use either `touch <path-to-directory> .env` or navigate to the directory and use `touch .env`
+   Note 1: In the above example you are trying to create an empty file called .env in each of the listed directories: backend and client. You can use either touch <path-to-directory> .env or navigate to the directory and use touch .env
```
Comment (trillium) [inline CONTRIBUTING.md]: Looks like you dropped the backticks by accident, they're needed to make sure that the commands show up separately in the text:

in current PR:
.env
touch <path-to-directory> .env

Prior to PR:
`.env`
`touch <path-to-directory> .env`

**Review by trillium — APPROVED** [review]

**Pull request merged**